exporters/prometheus/src/test/java/io/opentelemetry/exporter/prometheus/SerializerTest.java

```diff
@@ -367,51 +366,25 @@ void openMetrics() {
         .isEqualTo(
             "# TYPE instrument_name counter\n"
                 + "# HELP instrument_name description\n"
-                + "instrument_name_total{kp=\"vp\"} 5.0 1633950672.000\n"
-                + "# TYPE instrument_name gauge\n"
-                + "# HELP instrument_name description\n"
-                + "instrument_name{kp=\"vp\"} 5.0 1633950672.000\n"
-                + "# TYPE instrument_name gauge\n"
-                + "# HELP instrument_name description\n"
-                + "instrument_name{kp=\"vp\"} 5.0 1633950672.000\n"
-                + "# TYPE instrument_name gauge\n"
-                + "# HELP instrument_name description\n"
-                + "instrument_name{kp=\"vp\"} 5.0 1633950672.000\n"
-                + "# TYPE instrument_name counter\n"
-                + "# HELP instrument_name description\n"
-                + "instrument_name_total{kp=\"vp\"} 5.0 1633950672.000\n"
-                + "# TYPE instrument_name gauge\n"
-                + "# HELP instrument_name description\n"
-                + "instrument_name{kp=\"vp\"} 5.0 1633950672.000\n"
-                + "# TYPE instrument_name gauge\n"
-                + "# HELP instrument_name description\n"
-                + "instrument_name{kp=\"vp\"} 5.0 1633950672.000\n"
-                + "# TYPE instrument_name gauge\n"
-                + "# HELP instrument_name description\n"
-                + "instrument_name{kp=\"vp\"} 5.0 1633950672.000\n"
-                + "# TYPE instrument_name gauge\n"
-                + "# HELP instrument_name description\n"
-                + "instrument_name{kp=\"vp\"} 5.0 1633950672.000\n"
-                + "# TYPE instrument_name gauge\n"
-                + "# HELP instrument_name description\n"
-                + "instrument_name{kp=\"vp\"} 5.0 1633950672.000\n"
-                + "# TYPE instrument_name summary\n"
-                + "# HELP instrument_name description\n"
-                + "instrument_name_count{kp=\"vp\"} 5.0 1633950672.000\n"
-                + "instrument_name_sum{kp=\"vp\"} 7.0 1633950672.000\n"
-                + "instrument_name{kp=\"vp\",quantile=\"0.9\"} 0.1 1633950672.000\n"
-                + "instrument_name{kp=\"vp\",quantile=\"0.99\"} 0.3 1633950672.000\n"
-                + "# TYPE instrument_name histogram\n"
-                + "# HELP instrument_name description\n"
-                + "instrument_name_count{kp=\"vp\"} 2.0 1633950672.000\n"
-                + "instrument_name_sum{kp=\"vp\"} 1.0 1633950672.000\n"
-                + "instrument_name_bucket{kp=\"vp\",le=\"+Inf\"} 2.0 1633950672.000 # {span_id=\"0000000000000002\",trace_id=\"00000000000000000000000000000001\"} 4.0 0.001\n"
-                + "# TYPE instrument_name gauge\n"
-                + "# HELP instrument_name description\n"
+                + "instrument_name_total{type=\"mcds\"} 5.0 1633950672.000\n"
```

Comment:
Not related to this specific PR, but the spec [requires](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/metrics/datamodel.md#metric-metadata-1) that unit be appended to the metric name as a suffix. I don't believe we do that anywhere. I believe that would represent a breaking behavior change for prometheus users as their metric names would change when unit is present. 

Still, would be good to do that soon to minimize the impact.

Reply:
Thanks for noticing this - filed https://github.com/open-telemetry/opentelemetry-specification/issues/2497